q2_vizard/assets/lineplot/spec.json

```diff
@@ -0,0 +1,206 @@
+      "value": {"{{REPLACE_PARAM}}": "y_measure"}
+    }
+  ],
+
+  "data": [
+    {
+      "name": "scatter_table",
+      "values": {"{{REPLACE_PARAM}}": "metadata"}
+    },
+    {
+      "name": "line_table",
+      "values": {"{{REPLACE_PARAM}}": "averaged_metadata"}
+    }
+  ],
+
+  "scales": [
+    {
+      "name": "x_scale",
+      "type": "linear",
+      "range": "width",
+      "nice": true,
+      "zero": true,
+      "domain": {
+        "data": "scatter_table",
+        "field": {"{{REPLACE_PARAM}}": "x_measure"}
+      }
+    },
+    {
+      "name": "y_scale",
+      "type": "linear",
+      "range": "height",
+      "nice": true,
+      "zero": true,
+      "domain": {
+        "data": "scatter_table",
+        "field": {"{{REPLACE_PARAM}}": "y_measure"}
+      }
+    },
+    {
+      "name": "color_scale",
+      "type": "ordinal",
+      "range": "category",
+      "domain": {
+        "data": "line_table",
+        "field": {"{{REPLACE_PARAM}}": "facet_by"},
+        "sort": true
+      }
+    }
+],
+
+"axes": [
+    {
+      "scale": "x_scale",
+      "labelFontSize": 12,
+      "grid": true,
+      "orient": "bottom",
+      "tickCount": 5,
+      "title": {"{{REPLACE_PARAM}}": "x_measure"},
+      "titleFontSize": 14,
+      "titlePadding": 20
+    },
+    {
+      "scale": "y_scale",
+      "labelFontSize": 12,
+      "grid": true,
+      "orient": "left",
+      "title": {"{{REPLACE_PARAM}}": "y_measure"},
+      "titleFontSize": 14,
+      "titlePadding": 20
+    }
+  ],
+
+  "legends": [
+    {
+      "stroke": "color_scale",
+      "symbolSize": 150,
+      "symbolStrokeWidth": 2.5,
+      "labelFontSize": 14,
+      "title": {"{{REPLACE_PARAM}}": "facet_by"},
+      "titleFontSize": 16,
+      "encode": {
+        "symbols": {
+          "name": "legendSymbol",
+          "interactive": false
+        },
+        "labels": {
+          "name": "legendLabel",
+          "interactive": false
+        }
+      }
+    }
+  ],
+
+  "marks": [
+    {
+      "type": "group",
+      "from": {
+        "facet": {
+          "name": "lines",
+          "data": "line_table",
+          "groupby": {"{{REPLACE_PARAM}}": "facet_by"}
+        }
+      },
+      "marks": [
+        {
+          "type": "line",
+          "from": {"data": "lines"},
+          "encode": {
+            "update": {
+              "x": {
+                "scale": "x_scale",
+                "field": {"{{REPLACE_PARAM}}": "x_measure"}
+              },
+              "y": {
+                "scale": "y_scale",
+                "field": {"{{REPLACE_PARAM}}": "y_measure"}},
+              "interpolate": {"signal": "lineType"},
+              "stroke": {
+                "scale": "color_scale",
+                "field": {"{{REPLACE_PARAM}}": "facet_by"}
+              },
+              "strokeWidth": {"value": 2.5},
+              "strokeOpacity": {"value": 1}
+            }
+          }
+        },
+        {
+          "type": "symbol",
```

Comment:
This set can move just outside of the group marks section and in the global mark section.

Reply:
ahh, this is helpful - thanks! i didn't realize i'd done that - must have gotten my indentation mixed up 😅